spaceship.zsh

```diff
@@ -12,12 +12,12 @@ export SPACESHIP_VERSION='4.7.0'
 
 # Determination of Spaceship working directory
 # https://git.io/vdBH7
-if [[ -z "$SPACESHIP_ROOT" ]]; then
+spaceship::root() {
```

Comment:
I just realized that it can work even without this function. Just put the content of this function inside the condition.